toxic_timeout.go

```diff
@@ -8,6 +11,10 @@ type TimeoutToxic struct {
 	Enabled bool `json:"enabled"`
 	// Times in milliseconds
 	Timeout int64 `json:"timeout"`
+	// If true, use Toxicity
+	SometimesToxic bool `json:"sometimesToxic"`
+	// 'Toxicity' or probability of timing out 0..1
+	Toxicity float32 `json:"toxicity"`
```

Comment:
Yeah exactly. Maybe that's an OK compromise for Toxiproxy 1.x before we have our own `toxicity` field to be forwards compatible. Then we can simply remove this logic in 2.x and it'll be backwards compatible.